api/src/main/java/org/apache/iceberg/AddedRowsScanTask.java

```diff
@@ -55,7 +56,7 @@ default ChangelogOperation operation() {
 
   @Override
   default long sizeBytes() {
-    return length() + deletes().stream().mapToLong(ContentFile::fileSizeInBytes).sum();
+    return length() + ContentFileUtil.contentSizeInBytes(deletes());
```

Comment:
This logic is in `api`. I initially moved `ContentFileUtil` (which holds relevant logic) to `api`, which required moving more classes. Specifically, I am talking about `Partitioning` and `MetadataColumns`. I am thinking about alternatives. I could add a brand new `ScanTaskUtil` in `api`, but it is a bit awkward to have both. 

Any thoughts?

Reply:
This seems like a fairly simple method that doesn't need to require extensive refactoring. Why not add a `DeleteFileUtil` or something with this logic?

Reply:
I can. We will have `ContentFileUtil` and `DeleteFileUtil`. A bit awkward but not a big deal.